cabal.project

```diff
@@ -0,0 +1,2 @@
+
+packages: ./ghc-events.cabal
```

Comment:
Why do we need this file? I thought cabal would work without it given it has only the default package list.

Reply:
The cabal elders decided that cabal would scan from the current directory up to `/` for a cabal.project file if there is non in the current directory.

This means if there is any folder with such a file between `.` and `/` cabal will default to using that one instead of falling back to the .cabal file in the current file causing nonsensical errors.

This means the only way to ensure cabal works as expected in the current directory, is to make sure the current directory contains a cabal.project file.

Reply:
I didn't know that. Thanks for the info.